starwars-characters/src/characters.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Data from './characters.json';
+import Character from './character';
+import { BrowserRouter as Router, Route, Link} from 'react-router-dom';
+<<<<<<< HEAD
+=======
+
+function Char(){
+    return <div>{<Character />}</div>
+  }
+>>>>>>> 38787a7569a361f477bad6de8212013d9a30e9ac
+
+
+function Char(){
+    return <Character  />
+  }
+ 
+class CharacterList extends React.Component{
+    
+    render(){
+       
+        return(
+            <Router>
```

Comment:
You don't need a second router and switch to work, the base switch in App is enough. this will only make things more complicated.